bin/cml-publish.test.js

```diff
@@ -17,9 +17,17 @@ describe('CML e2e', () => {
         --gitlab-uploads  Uses GitLab uploads instead of CML storage. Use GitLab
```

Comment:
`expect(output).toMatchInlineSnapshot(...)` - it might trigger false alarms if library will change.
It is not a problem now, but it worth changing in the future in a more reliable way.

Reply:
by the library I mean `yargs`

Reply:
Yep, every time that we change the specs we have to update the template. It's a common practice in web dev to control that everything is looking as it should.

Reply:
I mean that it can change even without text changes - from the lib change (they might decide to change the default formatting).